### PR TITLE
Prepare for Marketplace VQA

### PR DESCRIPTION
**Pull request by dherault**
Before:
image
After:
image

### CODE REVIEW

File: src/components/RepositoryCard.tsx [hunk 1]
```diff
@@ -132,6 +133,8 @@ ref: Ref<any>) {
                     marginLeft={8}
                     paddingHorizontal={8}
                     paddingVertical={8}
+                    backgroundColor="transpanrent"
```
Comment (maciaszczykm) [inline src/components/RepositoryCard.tsx:136]: There is a typo.

Comment (dherault) [reply src/components/RepositoryCard.tsx:136]: Still works ;)

Comment (maciaszczykm) [reply src/components/RepositoryCard.tsx]: I think you can actually replace it with pure `Div` or `Flex` so it will not be needed anymore.

Comment (dherault) [reply src/components/RepositoryCard.tsx]: The Chip has the size="large" prop and aligns with the Figma file that has such a container.

**Pull request merged**